data/bangs.json

```diff
@@ -42005,15 +42005,15 @@
     "s": "Harvard University Library",
     "d": "hollis.harvard.edu",
     "t": "harvardlib",
-    "u": "https://hollis.harvard.edu/primo_library/libweb/action/dlSearch.do?institution=HVD&vid=HVD&tab=everything&displayField=all&search_scope=everything&mode=Basic&onCampus=false&displayMode=full&query=any%2Ccontains%2C{{{s}}}",
+    "u": "https://hollis.harvard.edu/primo_library/libweb/action/dlSearch.do?institution=HVD&vid=HVD&tab=everything&displayField=all&search_scope=everything&mode=Basic&onCampus=false&displayMode=full&query=any,contains,{{{s}}}",
     "c": "Research",
     "sc": "Academic"
   },
   {
     "s": "Twitter Hashtags",
     "d": "twitter.com",
     "t": "hashtag",
-    "u": "https://twitter.com/search?q=%23{{{s}}}",
```

Comment:
This URL template requires `#` to be encoded so it is not interpreted as a fragment identifier.